R/utils.R

```diff
@@ -842,6 +851,16 @@ wb_dims <- function(..., select = NULL) {
     }
   }
 
+  # Correct for empty data frame?
+  # patch so that we will want to span at least 1 row and one col
+  if (identical(nrow_to_span, 0L)) {
+    nrow_to_span <- 1L
+  }
+
+  if (identical(ncol_to_span, 0L)) {
+    ncol_to_span <- 1L
+  }
+
   row_span <- frow + seq_len(nrow_to_span) - 1L
```

Comment:
If `nrow_to_span` was 0, then `seq_len(0)` gave integer(0) frow +integer(0) = integer(0), so the fix just makes sure we span at least one column,

it could also be ncol_to_span <- min(1L, ncol_to_span), would be clearer?

Reply:
It’s fine for me